apps/ledger-live-desktop/src/renderer/Default.tsx

```diff
@@ -252,8 +264,12 @@ export default function Default() {
                   <USBTroubleshooting onboarding={!hasCompletedOnboarding} />
                 </Suspense>
               </Route>
+
               {!hasCompletedOnboarding ? (
-                <Route path="/settings" render={withSuspense(WelcomeScreenSettings)} />
+                <Switch>
+                  <Route path="/settings" render={withSuspense(WelcomeScreenSettings)} />
+                  <Route path="/recover/:appId" render={withSuspense(RecoverPlayer)} />
```

Comment:
Should we add the `FeatureToggle` used on the other `RecoverPlayer` ?
```suggestion
                  <FeatureToggle featureId="protectServicesDesktop">
                    <Route path="/recover/:appId" render={withSuspense(RecoverPlayer)} />
                  </FeatureToggle>
```